diff -urN rust/repo/examples/default/output/repository/Cargo.toml rust/repo/examples/crate/output/repository/Cargo.toml
--- rust/repo/examples/default/output/repository/Cargo.toml
+++ rust/repo/examples/crate/output/repository/Cargo.toml
@@ -1,23 +1,22 @@
-[workspace]
-members = [
-    "crate/name",
-]
-
-# We need to explicitly specify resolver 2.
-# We shouldn't have to per https://doc.rust-lang.org/edition-guide/rust-2021/default-cargo-resolver.html, but that does not appear to be the case
-resolver = "2"
-
-[workspace.package]
+[package]
+name = "mc-crate-name"
+version = "0.1.0"
 authors = ["MobileCoin"]
+# See https://crates.io/category_slugs for valid categories
+categories = ["cryptography", "no-std"]
+description = "A brief summary of the crate"
 edition = "2021"
+# See https://crates.io/keywords for the common keywords
+keywords = ["blockchain", "serde"]
 license = "Apache-2.0"
+readme = "README.md"
 repository = "https://github.com/mobilecoinfoundation/repository"
 rust-version = "1.67.1"
 
 [profile.release]
 lto = "thin"
 
-[workspace.metadata.release]
+[metadata.release]
 shared-version = true
 dev-version-ext = "beta.0"
 consolidate-commits = true
@@ -31,3 +30,5 @@
 pre-release-replacements = [
     {file="README.md", search="mc-[a-z-]+/[0-9.]+", replace="{{crate_name}}/{{version}}"},
 ]
+
+[dependencies]
\ No newline at end of file
diff -urN rust/repo/examples/default/output/repository/crate/name/Cargo.toml rust/repo/examples/crate/output/repository/crate/name/Cargo.toml
--- rust/repo/examples/default/output/repository/crate/name/Cargo.toml
+++ rust/repo/examples/crate/output/repository/crate/name/Cargo.toml
@@ -1,16 +0,0 @@
-[package]
-name = "mc-crate-name"
-version = "0.1.0"
-authors = { workspace = true }
-# See https://crates.io/category_slugs for valid categories
-categories = ["cryptography", "no-std"]
-description = "A brief summary of the crate"
-edition = { workspace = true }
-# See https://crates.io/keywords for the common keywords
-keywords = ["blockchain", "serde"]
-license = { workspace = true }
-readme = "README.md"
-repository = { workspace = true }
-rust-version = { workspace = true }
-
-[dependencies]
diff -urN rust/repo/examples/default/output/repository/crate/name/README.md rust/repo/examples/crate/output/repository/crate/name/README.md
--- rust/repo/examples/default/output/repository/crate/name/README.md
+++ rust/repo/examples/crate/output/repository/crate/name/README.md
@@ -1,21 +0,0 @@
-# MobileCoin: A brief summary of the crate
-
-[![Project Chat][chat-image]][chat-link]<!--
--->![License][license-image]<!--
--->![Architecture: any][arch-image]<!--
--->[![Crates.io][crate-image]][crate-link]<!--
--->[![Docs Status][docs-image]][docs-link]<!--
--->[![Dependency Status][deps-image]][deps-link]
-
-A brief summary of the crate
-
-[chat-image]: https://img.shields.io/discord/844353360348971068?style=flat-square
-[chat-link]: https://discord.gg/mobilecoin
-[license-image]: https://img.shields.io/crates/l/mc-crate-name?style=flat-square
-[arch-image]: https://img.shields.io/badge/arch-any-brightgreen?style=flat-square
-[crate-image]: https://img.shields.io/crates/v/mc-crate-name.svg?style=flat-square
-[crate-link]: https://crates.io/crates/mc-crate-name
-[docs-image]: https://img.shields.io/docsrs/mc-crate-name?style=flat-square
-[docs-link]: https://docs.rs/crate/mc-crate-name
-[deps-image]: https://deps.rs/crate/mc-crate-name/0.1.0/status.svg?style=flat-square
-[deps-link]: https://deps.rs/crate/mc-crate-name/0.1.0
diff -urN rust/repo/examples/default/output/repository/crate/name/src/lib.rs rust/repo/examples/crate/output/repository/crate/name/src/lib.rs
--- rust/repo/examples/default/output/repository/crate/name/src/lib.rs
+++ rust/repo/examples/crate/output/repository/crate/name/src/lib.rs
@@ -1,12 +0,0 @@
-// Copyright (c) 2023 The MobileCoin Foundation
-
-#![doc = include_str!("../README.md")]
-#![deny(missing_docs, missing_debug_implementations, unsafe_code)]
-
-#[cfg(test)]
-mod tests {
-    #[test]
-    fn it_works() {
-        assert_eq!(2 + 2, 4);
-    }
-}
diff -urN rust/repo/examples/default/output/repository/README.md rust/repo/examples/crate/output/repository/README.md
--- rust/repo/examples/default/output/repository/README.md
+++ rust/repo/examples/crate/output/repository/README.md
@@ -1,17 +1,25 @@
-# MobileCoin: A brief summary of the workspace
+# MobileCoin: A brief summary of the crate
 
 [![Project Chat][chat-image]][chat-link]<!--
 -->![License][license-image]<!--
+-->![Architecture: any][arch-image]<!--
+-->[![Crates.io][crate-image]][crate-link]<!--
+-->[![Docs Status][docs-image]][docs-link]<!--
 -->[![Dependency Status][deps-image]][deps-link]<!--
 -->[![CodeCov Status][codecov-image]][codecov-link]<!--
 -->[![GitHub Workflow Status][gha-image]][gha-link]<!--
 -->[![Contributor Covenant][conduct-image]][conduct-link]
 
-A brief summary of the workspace
+A brief summary of the crate
 
 [chat-image]: https://img.shields.io/discord/844353360348971068?style=flat-square
 [chat-link]: https://discord.gg/mobilecoin
 [license-image]: https://img.shields.io/crates/l/mc-crate-name?style=flat-square
+[arch-image]: https://img.shields.io/badge/arch-any-brightgreen?style=flat-square
+[crate-image]: https://img.shields.io/crates/v/mc-crate-name.svg?style=flat-square
+[crate-link]: https://crates.io/crates/mc-crate-name
+[docs-image]: https://img.shields.io/docsrs/mc-crate-name?style=flat-square
+[docs-link]: https://docs.rs/crate/mc-crate-name
 [deps-image]: https://deps.rs/repo/github/mobilecoinfoundation/repository/status.svg?style=flat-square
 [deps-link]: https://deps.rs/repo/github/mobilecoinfoundation/repository
 [codecov-image]: https://img.shields.io/codecov/c/github/mobilecoinfoundation/repository/develop?style=flat-square
diff -urN rust/repo/examples/default/output/repository/src/lib.rs rust/repo/examples/crate/output/repository/src/lib.rs
--- rust/repo/examples/default/output/repository/src/lib.rs
+++ rust/repo/examples/crate/output/repository/src/lib.rs
@@ -0,0 +1,12 @@
+// Copyright (c) 2023 The MobileCoin Foundation
+
+#![doc = include_str!("../README.md")]
+#![deny(missing_docs, missing_debug_implementations, unsafe_code)]
+
+#[cfg(test)]
+mod tests {
+    #[test]
+    fn it_works() {
+        assert_eq!(2 + 2, 4);
+    }
+}
