diff -urN rust/workspace-crate/examples/default/output/crate/name/Cargo.toml rust/workspace-crate/examples/license-apache/output/crate/name/Cargo.toml
--- rust/workspace-crate/examples/default/output/crate/name/Cargo.toml
+++ rust/workspace-crate/examples/license-apache/output/crate/name/Cargo.toml
@@ -8,7 +8,7 @@
 edition = { workspace = true }
 # See https://crates.io/keywords for the common keywords
 keywords = ["blockchain", "serde"]
-license = { workspace = true }
+license = "Apache-2.0"
 readme = "README.md"
 repository = { workspace = true }
 rust-version = { workspace = true}
